src/components/Popup/Popup.tsx

```diff
@@ -126,16 +127,45 @@ export default class Popup extends AutoControlledComponent<Extendable<PopupProps
 
   private static isBrowserContext = isBrowser()
 
+  private clickSubscription = EventStackSubscription.empty()
```

Comment:
just as a counterargument, we might argue that as long as Popup supports controlled mode, there are all the necessary tools to customize Popup's behavior.

Please, don't get me wrong - personally, see this suggestion as being absolutely reasonable. At the same time, would like to keep this PR simple and targeted to specific thing that it is aimed to address, so that we would be able to decide on this prop later and introduce separate PR for that.

Reply:
Makes sense!

Reply:
Makes sense!